core/sql/executor/ExHbaseAccess.cpp

```diff
@@ -444,6 +444,9 @@ ExHbaseAccessTcb::ExHbaseAccessTcb(
   rows_.len = 0;
   colVal_.val = 0;
   colVal_.len = 0;
+  asyncCompleteRetryCount_ = 0;
+  asyncOperationTimeout_ = 2;
```

Comment:
A comment on how timeout increases will be helpful. You could tie this to Hbase client retry settings if you like that approach. Also maybe the max retry count could be configurable through cqd or hbase settings?